src/ex2/main.js

```diff
@@ -1,10 +1,86 @@
+import ItemManager from "./item_manager.js";
 // Implement the `Main` class here
+class Main {
+	constructor() {
+		this.itemManager = new ItemManager();
+	}
+	init() {
+		document
+			.getElementById("list-item-submit")
+			.addEventListener("click", async () => {
+				await this.validateInput();
+			});
+		document.getElementById("clear-btn").addEventListener("click", () => {
+			this.clearAll();
+		});
+	}
 
+	renderTaskList(itemArray) {
+		const tasksList = document.getElementById("list");
+		const empyState = document.getElementById("empty-state");
+		const clearBtn = document.getElementById("clear-btn");
+		const inputTxt = document.getElementById("list-item-input");
+		tasksList.innerHTML = "";
+		if (itemArray.length > 0) {
```

Comment:
since your "else" is very short its better to flip the order of the "if" and "else" to make the code more readable